sentry-kotlin-multiplatform/src/androidMain/kotlin/io/sentry/kotlin/multiplatform/SentryPlatformOptions.android.kt

```diff
@@ -0,0 +1,17 @@
+package io.sentry.kotlin.multiplatform
+
+import io.sentry.kotlin.multiplatform.extensions.toAndroidSentryOptionsCallback
+
+public actual typealias SentryPlatformOptions = io.sentry.android.core.SentryAndroidOptions
+
+internal actual fun SentryPlatformOptions.prepareForInit() {
+    sdkVersion?.name = BuildKonfig.SENTRY_KMP_ANDROID_SDK_NAME
+    sdkVersion?.version = BuildKonfig.VERSION_NAME
+}
```

Comment:
should we also move [these lines](https://github.com/getsentry/sentry-kotlin-multiplatform/pull/221/files#diff-14b13973931d1582b7429057cdb84ba6068f69479da997c3c8f39a7cff12459dR24-R26) over here?